web/src/utilities/utilities.js

```diff
@@ -611,6 +611,38 @@ export const Utilities = {
     }
   },
 
+  isInitialAppInstall(app) {
+    if (!app) {
+      return false;
```

Comment:
shouldn't this be true?

Reply:
seems reasonable to flip this. i did not manage to reproduce this case ever since the only caller right now enforces that `app` exists before calling this function.